src/google_apps_clj/google_sheets_v4.clj

```diff
@@ -238,3 +301,44 @@
                    (add-sheet-id))]
     (write-sheet service worksheet-id sheet-id table)
     sheet-id))
+
+(defn get-effective-vals
+  "sheet-ranges is a list of strings, using the A1 syntax, eg [\"Sheet!A1:Z9\"]
+  Returns a list of tables in corresponding to sheet-ranges.  Only one
+  sheet (tab) can be specified per batch, due to a quirk of Google's API as far
+  as we can tell."
+  [service worksheet-id sheet-ranges]
```

Comment:
I like the idea of having a more clojurey interface to the syntax. I am wondering if a use case we should consider is someone looking at the cells visually and wanting to script up a range to fetch w/o doing off-by-one math in their head. I'd lean towards 0-indexed like you suggest but I wonder if there is an argument for `[["A" 1] ["Z" 9]]` (not sure I'm convinced of this)